### PR TITLE
Re-introduce HV locking

### PR DESCRIPTION
**Pull request by TuxPowered42**

### CODE REVIEW

**Comment by TuxPowered42** [comment]
retest this please

**Comment by TuxPowered42** [comment]
retest this please

**Comment by TuxPowered42** [comment]
retest this please

**Comment by TuxPowered42** [comment]
retest this please

**Comment by TuxPowered42** [comment]
retest this please

**Comment by TuxPowered42** [comment]
retest this please

**Comment by TuxPowered42** [comment]
retest this please

**Comment by TuxPowered42** [comment]
retest this please

**Comment by TuxPowered42** [comment]
retest this please

File: igvm/settings.py [hunk 1]
```diff
@@ -77,7 +77,8 @@
 MAC_ADDRESS_PREFIX = (0xCA, 0xFE, 0x01)
 
 try:
-    IGVM_IMAGE_URL = environ['IGVM_IMAGE_URL']
+    # IGVM_IMAGE_URL = environ['IGVM_IMAGE_URL']
+    IGVM_IMAGE_URL = 'http://foreman-master.ndco.ig.local:8080/new_images/{image}'
```
Comment (HerrSpace) [inline igvm/settings.py]: If you are certain it works set it on puppet, not here.

File: tests/test_integration.py [hunk 2]
```diff
@@ -144,7 +144,7 @@ def setUp(self):
             'int:basejessie:stable',
             'int:innogames:stable jessie',
         ]
-        obj['puppet_environment'] = None
+        obj['puppet_environment'] = 'make_serveradmin_chroot_unsafe'
```
Comment (HerrSpace) [inline tests/test_integration.py]: Same

**Comment by TuxPowered42** [comment]
retest this please

**Comment by HerrSpace** [comment]
retest this please

**Pull request merged**